docusaurus.config.js

```diff
@@ -0,0 +1,134 @@
+'use strict';
+const path = require('path');
+
+const lightCodeTheme = require('prism-react-renderer/themes/github');
+const darkCodeTheme = require('prism-react-renderer/themes/dracula');
+
+/** @type {import('@docusaurus/types').DocusaurusConfig} */
+module.exports = {
+  title: 'graphql-js',
+  tagline: 'A reference implementation of GraphQL for JavaScript',
+  url: 'https://js.graphql.org',
```

Comment:
```suggestion
  url: 'https://graphql-js.graphql.org',
```
We currently host docs under https://graphql.org/graphql-js/ + `js.graphql.org` means we should have one site for all JS projects including GraphiQL & LSP.